fog/view/server/src/shard_responses_processor.rs

```diff
@@ -45,10 +45,17 @@ pub fn process_shard_responses(
     let mut view_store_uris_for_authentication = Vec::new();
     let mut new_query_responses = Vec::new();
 
-    for (shard, response) in shards_and_responses {
+    log::debug!(
+        logger,
+        "process_shard_responses called on {} shards_and_responses",
+        shards_and_responses.len()
+    );
+
+    for (_i, (shard, response)) in shards_and_responses.into_iter().enumerate() {
```

Comment:
:wrench: If `_i` is ignored it seems like this can probably go back to the way it was without using enumerate
```suggestion
    for (shard, response) in shards_and_responses {
```

Reply:
I just followed the linter suggestion, but that makes more sense.

Fixed.